[1,"\u001B[31m\u001B[91mERROR: The specified directory ('.') is not a Quarto project.\n"]
[1,"(If you have not specified a path, quarto will attempt to render the entire current directory as a project.)\n"]
[1,"Stack trace:\n"]
[1,"    at render (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:91812:19)\n"]
[1,"    at async Command.actionHandler (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:91975:32)\n"]
[1,"    at async Command.execute (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:8253:13)\n"]
[1,"    at async Command.parseCommand (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:8143:20)\n"]
[1,"    at async quarto (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133948:9)\n"]
[1,"    at async file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133978:9\n"]
[1,"    at async file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133830:20\n"]
[1,"    at async mainRunner (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133832:9)\n"]
[1,"    at async file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133969:5\u001B[31m\n"]
[1,"\u001B[39m"]
[1,"\u001B[1m\u001B[33mError\u001B[39m in `quarto::quarto_render()`:\u001B[22m\n"]
[1,"\u001B[1m\u001B[22m\u001B[33m!\u001B[39m Error running quarto CLI from R.\n"]
[1,"\u001B[1mCaused by error in `quarto::quarto_render()`:\u001B[22m\n"]
[1,"\u001B[1m\u001B[22m\u001B[31m✖\u001B[39m Error returned by quarto CLI.\n"]
[1,"  -----------------------------\n"]
[1,"  \u001B[91mERROR: The specified directory ('.') is not a Quarto project.\u001B[39m\n"]
[1,"  (If you have not specified a path, quarto will attempt to render the entire\n"]
[1,"  current directory as a project.)\n"]
[1,"  \n"]
[1,"  Stack trace:\n"]
[1,"  at render\n"]
[1,"  (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:91812:19)\n"]
[1,"  at async Command.actionHandler\n"]
[1,"  (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:91975:32)\n"]
[1,"  at async Command.execute\n"]
[1,"  (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:8253:13)\n"]
[1,"  at async Command.parseCommand\n"]
[1,"  (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:8143:20)\n"]
[1,"  at async quarto\n"]
[1,"  (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133948:9)\n"]
[1,"  at async\n"]
[1,"  file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133978:9\n"]
[1,"  at async\n"]
[1,"  file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133830:20\n"]
[1,"  at async mainRunner\n"]
[1,"  (file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133832:9)\n"]
[1,"  at async\n"]
[1,"  file:///Applications/RStudio.app/Contents/Resources/app/quarto/bin/quarto.js:133969:5\n"]
[1,"  \n"]
[1,"\u001B[1mCaused by error:\u001B[22m\n"]
[1,"\u001B[33m!\u001B[39m System command 'quarto' failed\n"]
[1,"Backtrace:\n"]
[1,"\u001B[90m     \u001B[39m▆\n"]
[1,"\u001B[90m  1. \u001B[39m├─.rs.sourceWithProgress(...)\n"]
[1,"\u001B[90m  2. \u001B[39m│ └─\u001B[1mbase\u001B[22m::eval(statements[[idx]], envir = globalenv())\n"]
[1,"\u001B[90m  3. \u001B[39m│   └─base::eval(statements[[idx]], envir = globalenv())\n"]
[1,"\u001B[90m  4. \u001B[39m├─\u001B[1m\u001B[94mquarto::quarto_render(\u001B[39m\u001B[22m\u001B[1m\u001B[94m)\u001B[39m\u001B[22m\n"]
[1,"\u001B[90m  5. \u001B[39m│ └─quarto:::quarto_run(...)\n"]
[1,"\u001B[90m  6. \u001B[39m│   ├─\u001B[1mbase\u001B[22m::withCallingHandlers(...)\n"]
[1,"\u001B[90m  7. \u001B[39m│   └─\u001B[1mprocessx\u001B[22m::run(...)\n"]
[1,"\u001B[90m  8. \u001B[39m│     └─throw(...)\n"]
[1,"\u001B[90m  9. \u001B[39m│       └─\u001B[1mbase\u001B[22m::signalCondition(cond)\n"]
[1,"\u001B[90m 10. \u001B[39m└─\u001B[1mquarto\u001B[22m (local) `<fn>`(`<systm___>`)\n"]
[1,"\u001B[90m 11. \u001B[39m  └─quarto:::wrap_quarto_error(e, args, .call = .call)\n"]
[1,"\u001B[90m 12. \u001B[39m    └─\u001B[1mcli\u001B[22m::cli_abort(msg, call = .call, parent = cnd, )\n"]
[1,"\u001B[90m 13. \u001B[39m      └─\u001B[1mrlang\u001B[22m::abort(...)\n"]
[2,"Execution halted\n"]
